content/docs/overview.md

```diff
@@ -305,14 +305,55 @@ for key, value in some_map {
     fmt.println(key, value);
 }
 ```
-The iterated values are *copies* and cannot be written to. The following idiom is useful for iterating over a container in a by-reference manner:
+
+The iteration variables are _copies_ of the data in the array or map, and cannot be assigned to. See below for a way to get around this.
+
+**Note:** When iterating through a `string`, the characters will be `rune`s and not bytes. `for in` assumes the string to be encoded as UTF-8, and decodes each rune as it goes.
+
+### The Underscore (a.k.a unnamed variables)
+
+In certain situations you need to name a variable, but have no intent to use it.
+
+Consider:
 ```odin
-for _, i in some_slice {
-    some_slice[i] = something;
+for value, index in some_slice {
+    some_slice[index] = something;
 }
 ```
+`value` is never used here.
+
+In instances like this, you may use `_`:
```

Comment:
In instances like this, you may use `_`, which is known as the blank identifier: